requirements.yml

```diff
@@ -1,8 +1,8 @@
 ---
-- src: https://github.com/redhat-nfvpe/ansible-role-install-go
+- src: https://github.com/gantsign/ansible-role-golang
```

Comment:
I think I'm OK with changing to this galaxy role this if you think this role is well maintained.

Reply:
It supports all major distros and it is actively maintained.